CenterForOpenScience/osf.io
# [Feature Fix] Fix fangorn tests and minor update [OSF-6319]

## Purpose
Fangorn tests needed to have updated names and added minor fixes along the way.

## Changes

Rename fangorn tests

## Side effects


1. Add check for no rows on multiselect
2. Disable all fangorn figshare item drag outs

## Ticket
[#OSF-6319]
https://openscience.atlassian.net/browse/OSF-6319


## QA Notes
1. Test that you ctrl-clicking when you have only a single item selected, and you select the time you are already on does not throw an error in the console
2. Check that attempting to drag figshare items (both published and draft) out of figshare shows disabled the drops
